src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/types/index.ts

```diff
@@ -219,7 +219,6 @@ export interface IRuntimeState {
 }
 
 export interface ISchemaValidator {
-  rootElement: any;
   rootElementPath: string;
   schema: any;
   validator: Ajv;
```

Comment:
This is now either Ajv or Ajv2020 now?

Reply:
Good catch. Actually refactored to use a shared base class for both cases as we only use general methods.